### ISSUE TITLE
(Python 3.11) Building error at installation

### ISSUE THREAD

**Issue by mxrch** [maxbachmann/Levenshtein#33]
Hello !
I always used your very nice library with Python 3.8 to 3.10, but when I try to install it with Python 3.11rc1 on Windows 11, I get this traceback : 
```python
PS C:\Users\thher> py -m pip install levenshtein
Collecting levenshtein
  Using cached Levenshtein-0.20.2.tar.gz (114 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting rapidfuzz<3.0.0,>=2.3.0
  Using cached rapidfuzz-2.6.1-cp311-cp311-win_amd64.whl (1.3 MB)
Collecting jarowinkler<2.0.0,>=1.2.0
  Using cached jarowinkler-1.2.1-cp311-cp311-win_amd64.whl (62 kB)
Building wheels for collected packages: levenshtein
  Building wheel for levenshtein (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for levenshtein (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [289 lines of output]
      Not searching for unused variables given on the command line.
      -- The C compiler identification is GNU 12.1.0
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: C:/Users/thher/gcc/bin/gcc.exe - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- The CXX compiler identification is GNU 12.1.0
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: C:/Users/thher/gcc/bin/c++.exe - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      CMake Error at CMakeLists.txt:9 (message):
        MSVC is required to pass this check.


      -- Configuring incomplete, errors occurred!
      See also "C:/Users/thher/AppData/Local/Temp/pip-install-hdi1djen/levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56/_cmake_test_compile/build/CMakeFiles/CMakeOutput.log".
      Not searching for unused variables given on the command line.
      CMake Error at CMakeLists.txt:2 (PROJECT):
        Generator

          Visual Studio 17 2022

        could not find any instance of Visual Studio.



      -- Configuring incomplete, errors occurred!
      See also "C:/Users/thher/AppData/Local/Temp/pip-install-hdi1djen/levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56/_cmake_test_compile/build/CMakeFiles/CMakeOutput.log".
      Not searching for unused variables given on the command line.
      -- The C compiler identification is GNU 12.1.0
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: C:/Users/thher/gcc/bin/gcc.exe - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- The CXX compiler identification is GNU 12.1.0
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: C:/Users/thher/gcc/bin/c++.exe - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      CMake Error at CMakeLists.txt:9 (message):
        MSVC is required to pass this check.


      -- Configuring incomplete, errors occurred!
      See also "C:/Users/thher/AppData/Local/Temp/pip-install-hdi1djen/levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56/_cmake_test_compile/build/CMakeFiles/CMakeOutput.log".
      Not searching for unused variables given on the command line.
      -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.22622.
      -- The C compiler identification is MSVC 19.29.30146.0
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- The CXX compiler identification is MSVC 19.29.30146.0
      CMake Warning (dev) at C:/Program Files/CMake/share/cmake-3.24/Modules/CMakeDetermineCXXCompiler.cmake:162 (if):
        Policy CMP0054 is not set: Only interpret if() arguments as variables or
        keywords when unquoted.  Run "cmake --help-policy CMP0054" for policy
        details.  Use the cmake_policy command to set the policy and suppress this
        warning.

        Quoted variables like "MSVC" will no longer be dereferenced when the policy
        is set to NEW.  Since the policy is not set the OLD behavior will be used.
      Call Stack (most recent call first):
        CMakeLists.txt:4 (ENABLE_LANGUAGE)
      This warning is for project developers.  Use -Wno-dev to suppress it.

      CMake Warning (dev) at C:/Program Files/CMake/share/cmake-3.24/Modules/CMakeDetermineCXXCompiler.cmake:183 (elseif):
        Policy CMP0054 is not set: Only interpret if() arguments as variables or
        keywords when unquoted.  Run "cmake --help-policy CMP0054" for policy
        details.  Use the cmake_policy command to set the policy and suppress this
        warning.

        Quoted variables like "MSVC" will no longer be dereferenced when the policy
        is set to NEW.  Since the policy is not set the OLD behavior will be used.
      Call Stack (most recent call first):
        CMakeLists.txt:4 (ENABLE_LANGUAGE)
      This warning is for project developers.  Use -Wno-dev to suppress it.

      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Configuring done
      -- Generating done
      -- Build files have been written to: C:/Users/thher/AppData/Local/Temp/pip-install-hdi1djen/levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56/_cmake_test_compile/build
      -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.22622.
      -- The C compiler identification is MSVC 19.29.30146.0
      -- The CXX compiler identification is MSVC 19.29.30146.0
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Found PythonInterp: C:/Users/thher/AppData/Local/Programs/Python/Python311/python.exe (found version "3.11")
      -- Found PythonLibs: C:/Users/thher/AppData/Local/Programs/Python/Python311/libs/python311.lib (found version "3.11.0rc1")
      -- Found Python: C:/Users/thher/AppData/Local/Programs/Python/Python311/python.exe (found version "3.11.0") found components: Interpreter Development Development.Module Development.Embed
      Using packaged version of rapidfuzz-cpp
      -- Performing Test Weak Link MODULE -> SHARED (gnu_ld_ignore) - Failed
      -- Performing Test Weak Link MODULE -> SHARED (osx_dynamic_lookup) - Failed
      -- Performing Test Weak Link MODULE -> SHARED (no_flag) - Failed
      _modinit_prefix:PyInit_
      -- Configuring done
      -- Generating done
      CMake Warning:
        Manually-specified variables were not used by the project:

          Python3_EXECUTABLE
          Python3_INCLUDE_DIR
          Python3_LIBRARY
          SKBUILD


      -- Build files have been written to: C:/Users/thher/AppData/Local/Temp/pip-install-hdi1djen/levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56/_skbuild/win-amd64-3.11/cmake-build
      Microsoft (R) Build Engine version 16.11.2+f32259642 pour .NET Framework
      Copyright (C) Microsoft Corporation. Tous droits r‚serv‚s.

      C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(517,5): warning MSB8029: Le r‚pertoire interm‚diaire ou le r‚pertoire de sortie ne peut pas se trouver sous le r‚pertoire temporaire car cela risque de cr‚er des problŠmes avec la g‚n‚ration incr‚mentielle. [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\ZERO_CHECK.vcxproj]
        Checking Build System
      C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(517,5): warning MSB8029: Le r‚pertoire interm‚diaire ou le r‚pertoire de sortie ne peut pas se trouver sous le r‚pertoire temporaire car cela risque de cr‚er des problŠmes avec la g‚n‚ration incr‚mentielle. [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
        Building Custom Rule C:/Users/thher/AppData/Local/Temp/pip-install-hdi1djen/levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56/src/Levenshtein/CMakeLists.txt
        levenshtein_cpp.cxx
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(543,1): warning C4267: '='ÿ: conversion de 'size_t' en 'long', perte possible de donn‚es [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(39): message : voir la r‚f‚rence … l'instanciation de la fonction modŠle 'auto lev_set_median::<lambda_505ef21b6c5a639ad1822147016390c0>::operator ()<uint8_t*,uint8_t*>(uint8_t *,uint8_t *) const' en cours de compilation [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(562): message : voir la r‚f‚rence … l'instanciation de la fonction modŠle 'auto visit<lev_set_median::<lambda_505ef21b6c5a639ad1822147016390c0>,>(const RF_String &,Func &&)' en cours de compilation [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
                with
                [
                    Func=lev_set_median::<lambda_505ef21b6c5a639ad1822147016390c0>
                ]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(552,34): warning C4244: '='ÿ: conversion de 'int64_t' en '_Ty', perte possible de donn‚es [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
                with
                [
                    _Ty=long
                ]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(2428,121): warning C4127: l'expression conditionnelle est une constante [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(2436,122): warning C4127: l'expression conditionnelle est une constante [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(2626,119): warning C4127: l'expression conditionnelle est une constante [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(2634,120): warning C4127: l'expression conditionnelle est une constante [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(2810,90): warning C4100: '__pyx_self'ÿ: paramŠtre formel non r‚f‚renc‚ [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(3042,96): warning C4100: '__pyx_self'ÿ: paramŠtre formel non r‚f‚renc‚ [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(3286,99): warning C4100: '__pyx_self'ÿ: paramŠtre formel non r‚f‚renc‚ [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(3530,94): warning C4100: '__pyx_self'ÿ: paramŠtre formel non r‚f‚renc‚ [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(3758,93): warning C4100: '__pyx_self'ÿ: paramŠtre formel non r‚f‚renc‚ [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(4043,94): warning C4100: '__pyx_self'ÿ: paramŠtre formel non r‚f‚renc‚ [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(5648,41): warning C4996: 'PyBytesObject::ob_shash': deprecated in 3.11 [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(5649,41): warning C4996: 'PyBytesObject::ob_shash': deprecated in 3.11 [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\levenshtein_cpp.cxx(7306,5): error C2027: utilisation du type non d‚fini '_frame' [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Programs\Python\Python311\Include\pytypedefs.h(22): message : voir la d‚claration de '_frame' [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
        _levenshtein.cpp
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(543,1): warning C4267: '='ÿ: conversion de 'size_t' en 'long', perte possible de donn‚es [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(39): message : voir la r‚f‚rence … l'instanciation de la fonction modŠle 'auto lev_set_median::<lambda_505ef21b6c5a639ad1822147016390c0>::operator ()<uint8_t*,uint8_t*>(uint8_t *,uint8_t *) const' en cours de compilation [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(562): message : voir la r‚f‚rence … l'instanciation de la fonction modŠle 'auto visit<lev_set_median::<lambda_505ef21b6c5a639ad1822147016390c0>,>(const RF_String &,Func &&)' en cours de compilation [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
                with
                [
                    Func=lev_set_median::<lambda_505ef21b6c5a639ad1822147016390c0>
                ]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(552,34): warning C4244: '='ÿ: conversion de 'int64_t' en '_Ty', perte possible de donn‚es [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
                with
                [
                    _Ty=long
                ]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.cpp(174,23): warning C4389: '!='ÿ: incompatibilit‚ signed/unsigned [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.hpp(39): message : voir la r‚f‚rence … l'instanciation de la fonction modŠle 'auto lev_quick_median::<lambda_3d674032857d8ac4d599d2228b77b1cc>::operator ()<uint8_t*,uint8_t*>(uint8_t *,uint8_t *) const' en cours de compilation [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.cpp(199): message : voir la r‚f‚rence … l'instanciation de la fonction modŠle 'auto visit<lev_quick_median::<lambda_3d674032857d8ac4d599d2228b77b1cc>,>(const RF_String &,Func &&)' en cours de compilation [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
                with
                [
                    Func=lev_quick_median::<lambda_3d674032857d8ac4d599d2228b77b1cc>
                ]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.cpp(183,23): warning C4389: '!='ÿ: incompatibilit‚ signed/unsigned [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
      C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\src\Levenshtein\Levenshtein-c\_levenshtein.cpp(195,23): warning C4389: '!='ÿ: incompatibilit‚ signed/unsigned [C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build\src\Levenshtein\levenshtein_cpp.vcxproj]
        G‚n‚ration de code en cours...
        File "C:\Users\thher\AppData\Local\Temp\pip-build-env-snm1r45x\overlay\Lib\site-packages\skbuild\setuptools_wrap.py", line 645, in setup
          cmkr.make(make_args, install_target=cmake_install_target, env=env)
        File "C:\Users\thher\AppData\Local\Temp\pip-build-env-snm1r45x\overlay\Lib\site-packages\skbuild\cmaker.py", line 680, in make
          self.make_impl(clargs=clargs, config=config, source_dir=source_dir, install_target=install_target, env=env)
        File "C:\Users\thher\AppData\Local\Temp\pip-build-env-snm1r45x\overlay\Lib\site-packages\skbuild\cmaker.py", line 704, in make_impl
          raise SKBuildError(


      --------------------------------------------------------------------------------
      -- Trying "Ninja (Visual Studio 17 2022 x64 v143)" generator
      --------------------------------
      ---------------------------
      ----------------------
      -----------------
      ------------
      -------
      --
      --
      -------
      ------------
      -----------------
      ----------------------
      ---------------------------
      --------------------------------
      -- Trying "Ninja (Visual Studio 17 2022 x64 v143)" generator - failure
      --------------------------------------------------------------------------------



      --------------------------------------------------------------------------------
      -- Trying "Visual Studio 17 2022 x64 v143" generator
      --------------------------------
      ---------------------------
      ----------------------
      -----------------
      ------------
      -------
      --
      --
      -------
      ------------
      -----------------
      ----------------------
      ---------------------------
      --------------------------------
      -- Trying "Visual Studio 17 2022 x64 v143" generator - failure
      --------------------------------------------------------------------------------



      --------------------------------------------------------------------------------
      -- Trying "Ninja (Visual Studio 16 2019 x64 v142)" generator
      --------------------------------
      ---------------------------
      ----------------------
      -----------------
      ------------
      -------
      --
      --
      -------
      ------------
      -----------------
      ----------------------
      ---------------------------
      --------------------------------
      -- Trying "Ninja (Visual Studio 16 2019 x64 v142)" generator - failure
      --------------------------------------------------------------------------------



      --------------------------------------------------------------------------------
      -- Trying "Visual Studio 16 2019 x64 v142" generator
      --------------------------------
      ---------------------------
      ----------------------
      -----------------
      ------------
      -------
      --
      --
      -------
      ------------
      -----------------
      ----------------------
      ---------------------------
      --------------------------------
      -- Trying "Visual Studio 16 2019 x64 v142" generator - success
      --------------------------------------------------------------------------------

      Configuring Project
        Working directory:
          C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build
        Command:
          cmake 'C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56' -G 'Visual Studio 16 2019' '-DCMAKE_INSTALL_PREFIX:PATH=C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-install' -DPYTHON_VERSION_STRING:STRING=3.11.0rc1 -DSKBUILD:INTERNAL=TRUE '-DCMAKE_MODULE_PATH:PATH=C:\Users\thher\AppData\Local\Temp\pip-build-env-snm1r45x\overlay\Lib\site-packages\skbuild\resources\cmake' '-DPython3_EXECUTABLE:FILEPATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\python.exe' '-DPython3_INCLUDE_DIR:PATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\Include' '-DPython3_LIBRARY:PATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\libs\python311.lib' '-DPython_EXECUTABLE:FILEPATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\python.exe' '-DPython_INCLUDE_DIR:PATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\Include' '-DPython_LIBRARY:PATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\libs\python311.lib' '-DPYTHON_EXECUTABLE:FILEPATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\python.exe' '-DPYTHON_INCLUDE_DIR:PATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\Include' '-DPYTHON_LIBRARY:PATH=C:\Users\thher\AppData\Local\Programs\Python\Python311\libs\python311.lib' -T v142 -A x64 -DCMAKE_BUILD_TYPE:STRING=Release

      Traceback (most recent call last):

      An error occurred while building with CMake.
        Command:
          cmake --build . --target install --config Release --
        Install target:
          install
        Source directory:
          C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56
        Working directory:
          C:\Users\thher\AppData\Local\Temp\pip-install-hdi1djen\levenshtein_8bd27ecdfaa145b1ac36fc5dbf00ff56\_skbuild\win-amd64-3.11\cmake-build
      Please check the install target is valid and see CMake's output for more information.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for levenshtein
Failed to build levenshtein
ERROR: Could not build wheels for levenshtein, which is required to install pyproject.toml-based projects
```

**Comment by maxbachmann** [comment]
I forgot to release a new version with Python 3.11 support when I released the update for rapidfuzz.
I just released a new version `0.20.3` which adds support for Python 3.11.

**Comment by mxrch** [comment]
Thanks for the quick action, it's all good now ! :D 
```powershell
PS C:\Users\thher> py -V
Python 3.11.0rc1
PS C:\Users\thher> py -m pip install levenshtein
Could not find platform dependent libraries <exec_prefix>
Collecting levenshtein
  Downloading Levenshtein-0.20.3-cp311-cp311-win_amd64.whl (182 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 182.9/182.9 kB 2.2 MB/s eta 0:00:00
Collecting rapidfuzz<3.0.0,>=2.3.0
  Using cached rapidfuzz-2.6.1-cp311-cp311-win_amd64.whl (1.3 MB)
Collecting jarowinkler<2.0.0,>=1.2.0
  Using cached jarowinkler-1.2.1-cp311-cp311-win_amd64.whl (62 kB)
Installing collected packages: jarowinkler, rapidfuzz, levenshtein
Successfully installed jarowinkler-1.2.1 levenshtein-0.20.3 rapidfuzz-2.6.1
```
